event_sale_session/README.rst

```diff
@@ -77,6 +77,10 @@ Contributors
 
     * Iván Todorovich <ivan.todorovich@gmail.com>
 
+* `GutierrezTi Team <https://gutierrezti.es>`_
+    * Joaquín Gutiérrez Pedrosa <joaquin@gutierrezti.es>
```

Comment:
This file is auto-generated from the ones in the `readme` directory.
Changes done here will be overwritten.

Instead, update the `readme/CONTRIBUTORS.rst` file